obs-packaging/build_from_docker.sh

```diff
@@ -26,7 +26,9 @@ source "${script_dir}/scripts/obs-docker.sh"
 GO_ARCH=$(go env GOARCH)
 export GO_ARCH
 
-OSCRC="${HOME}/.oscrc"
+# oscrc exists at different places on different distros
+[ -f "${HOME}/.config/osc/oscrc" ] && OSCRC="${HOME}/.config/osc/oscrc"
+[ -f "${HOME}/.oscrc" ] && OSCRC="${HOME}/.oscrc"
```

Comment:
Hi, this would not work for users not already having a `oscrc` file on their system.

So  I think the correct way to do it is:
```
[ -f "${HOME}/.oscrc" ] && OSCRC="${HOME}/.oscrc"
OSCRC="${HOME}/.config/osc/oscrc"
```
And also add a `mkdir -p $(dirname $OSCRC)` when creating the file few lines below.

Reply:
updated. thanks!